plan/plan.go

```diff
@@ -52,6 +52,9 @@ type Plan interface {
 	// findColumn finds the column in basePlan's schema.
 	// If the column is not in the schema, returns error.
 	findColumn(*ast.ColumnName) (*expression.Column, int, error)
+
+	// ResolveIndices resolves the indices for columns. After doing this, the columns can evaluate the rows by their indices.
+	ResolveIndices()
```

Comment:
why move this.

Reply:
Insert/Delete/Update also needs resolveIndices